src/en/tutorial-client-basic.md

```diff
@@ -0,0 +1,367 @@
+2        started  10.243.67.141  juju-ded876-2        bionic      Running
+```
+
+The output is broken up into four sections.
+
+The **top** section mentions basic information such as the names of the current
+model and controller ('alpha' and 'lxd' respectively), followed by the cloud
+name ('localhost'), what version is running on the agents in the current model,
+whether Juju is being used in a third-party context (see
+[Managed solutions][experts-sla]) and finally the timestamp of the current
+controller.
+
+The **App** section contains information at the application level. It is
+closely related to the providence of an application's charm. The 'Scale' tells
+us how many units exist for an application while the 'Rev' column shows the
+charm's revision number.
+
+The **Unit** section contains information at the unit level. It lists them,
+along with information that is passed back from the *unit agent*. The type of
+data available is is very charm-specific. For instance, the 'unknown' workload
+message for the apache2 units is not necessarily a bad thing. It's just that
+the associated charm was not written to give something more insightful.
+
+The **Machine** section contains information at the machine level. It lists
+machines by their ID (e.g. '0/lxd/0' or just '1'). The 'Inst id' is the
+instance id of the machine that gets passed to the cloud provider as the
+instance name.
+
+Notice how the `machines` command output is a subset of the `status` command
+output.
+
+### Inspecting the logs
+
+Juju logs are inspected on a per-model basis using a specific utility.
+
+To view (tail) the logs of the 'alpha' model:
+
+```bash
+juju debug-log -m alpha
+```
+
+There are a number of ways to configure the behaviour of the `debug-log`
+command.
+
+### Connecting to a machine via SSH
+
+The system user who created the controller (`bootstrap` command) will
+automatically have SSH access to all machines in the original two default
+models ('controller' and 'default') and any models they create afterwards.
+
+To connect to a machine simply refer to the machine ID (from the `machines`
+command). Here we connect to machine '1' in the current model:
+
+```bash
+juju ssh 1
+```
+
+This command will always get you to the controller machine:
+
+```bash
+juju ssh -m controller 0
+```
+
+### Removing an application
+
+To remove the 'apache2' application, including all units, along with associated
+machines (provided they are not hosting another application's units).
+
+```bash
+juju remove-application apache2
+```
+
+### Removing a unit
+
+To remove just a single unit, such as 'apache2/1':
+
+```bash
+juju remove-unit apache2/1
+```
+
+Like the `remove-application` command, this command will also remove the
+machine if it is now devoid of units.
+
+### Removing a machine
+
+To remove the machine whose ID is '1':
+
+```bash
+juju remove-machine 1
+```
+
+Note that it is not possible to remove a machine (without the `--force` option)
```

Comment:
This reads that the recommended way to remove a machine with a unit on it is to go the --force route. I'd prefer we noted that if you remove a unit on a machine and it's the last unit on that machine the machine will automatically be cleaned up.

Reply:
which to be fair you note above in remove-application but this section still reads a bit wrong imo. I'd just leave out the note about the error perhaps or call out that if you have a unit on it to use remove-unit or application instead.